### ISSUE TITLE
very serious blind spot from AI competition 2024

### ISSUE THREAD

**Issue by yehu3d** [lightvector/KataGo#1003]
![捕获](https://github.com/user-attachments/assets/ed2d59d5-6e30-40fe-86e2-4c9a9f6092b6)
chinese rules komi 7.0 button=true.

**Comment by yehu3d** [comment]
dont support ,so i upload txt ,download it and rename it from 2.txt to 2.sfg
[2.txt](https://github.com/user-attachments/files/17881982/2.txt)

**Comment by michito744** [comment]
Results verified with Chinese rules
![2024-11-24](https://github.com/user-attachments/assets/b2bc655b-40c6-4983-be8e-5b8200830da2)

Results verified with Japanese rules
![2024-11-24 (1)](https://github.com/user-attachments/assets/885ee0c3-5ee7-4cf4-846a-873399d26acb)

Conclusion: Could not reproduce

**Comment by michito744** [comment]
@Oh sorry,

Very inefficient and a waste of energy, but it can converge on the right conclusion.
![2024-11-25 (11)](https://github.com/user-attachments/assets/0b4bed96-786e-4f92-9f78-87c91c0b211a)

If it cannot, config is not properly set up.

**Comment by yehu3d** [comment]
@michito744 Yes, starting from this position, if you continue to calculate, you can arrive at the correct answer. However, if you start calculating from an earlier position (for example, 10 moves ago), it might require significantly more computations to reach the correct answer. This could be considered a "blind spot."

**Comment by michito744** [comment]
@yehu3d Black is only winning by earning 0.5 points by “first pass” in a situation where he has the advantage of ko, so there is no other way to increase the percentage of “pass” explored, which are rarely explored in normal circumstances.

(And its very very inefficient and a waste of energy.)
